components/script/script_thread.rs

```diff
@@ -645,6 +646,10 @@ pub struct ScriptThread {
 
     /// FIXME(victor):
     webrender_api_sender: RenderApiSender,
+
+    // TODO pylbrecht
+    // write meaningful docstring
+    layout_thread_is_busy: Arc<AtomicBool>,
```

Comment:
We probably do not need to store this in ScriptThread?

Reply:
Well, the busy flag is created in `UnprivilegedPipelineContent::start_all()` and passed via `clone()` to the layout and script thread. I'm not quite sure how to retain the relationship between the two threads, when creating `Window`s in `ScriptThread::load()`, unless storing the busy flag in `ScriptThread` itself.

Reply:
You're right, but I think that we can store it as a member of InProgressLoad instead.